### PR TITLE
Fix escaping

### PR DESCRIPTION
**Pull request by viddo**
ping @javisantana

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4675/))

**Comment by javisantana** [comment]
@xavijam please, take a look

File: lib/assets/javascripts/cartodb/organization/views/quota_dropdown.jst.ejs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="UserQuotaDropdown">
-  <h4 class="UserQuotaDropdown-title">Change <%= userName %> quota</h4>
+  <h4 class="UserQuotaDropdown-title">Change <%- username %> quota</h4>
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/organization/views/quota_dropdown.jst.ejs]: "userName" !== "username"

**Comment by xavijam** [comment]
Apart from ^^^, :+1:.

File: lib/assets/javascripts/cartodb/organization/views/quota_dropdown.jst.ejs (hunk 1 above)
Comment (viddo) [inline lib/assets/javascripts/cartodb/organization/views/quota_dropdown.jst.ejs]: ouch, good one

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4682/))

**Pull request merged**